shared/teams/team/tabs/index.tsx

```diff
@@ -36,6 +36,7 @@ const TeamTabs = (props: TeamTabsProps) => {
     // TODO: should we not show bots if there are no bots and you have no permissions?
     {title: 'bots' as const},
     ...(props.numSubteams > 0 || props.showSubteams ? [{title: 'subteams' as const}] : []),
+    {title: 'emoji' as const},
```

Comment:
Probably shouldn't add this on mobile until it gets a dev pass. Tabs need to scroll, header titles all overlap.